### ISSUE TITLE
awesome-typescript-loader does not obey experimentalDecorators setting in tsconfig.json

### ISSUE THREAD

**Issue by LukeNZ** [s-panferov/awesome-typescript-loader#198]
# Issue

Running `webpack` using the below configuration results in the following **erro**r being emitted for each Angular 2 component:

`ERROR in [default] C:\app\components\app.component.ts:17:13`
` Experimental support for decorators is a feature that is subject to change in a future release. Set the 'experimentalDecorators' option to remove this warning. `

This does not occur when using the native TypeScript compiler in Visual Studio, or using `ts-loader` (which is much slower). 

# Environment:

- Visual Studio 2015 Update 3
- TypeScript 2.0.0-beta
- Webpack 1.13.1
- awesome-typescript-loader 2.1.0

# Webpack configuration

```
var webpack = require('webpack');
var CopyWebpackPlugin = require('copy-webpack-plugin');
var path = require('path');

module.exports = {
    context: path.join(__dirname, '/'),
    entry: {
        'polyfills': './Scripts/app/polyfills.ts',
        'vendor': './Scripts/app/vendor.ts',
        'app': './Scripts/app/main.ts'
    },
    resolve: {
        extensions: ['', '.js', '.ts']
    },
    output: {
        path: path.join(__dirname, 'wwwroot/angular'),
        publicPath: '/wwwroot/angular',
        filename: '[name].bundle.js'
    },
    module: {
        loaders: [
            {
                test: /\.ts$/,
                loader: 'awesome-typescript-loader'
            }
        ]
    },
    plugins: [
        new webpack.optimize.CommonsChunkPlugin({
            name: ['app', 'vendor', 'polyfills']
        })
    ]
};
```

# tsconfig.json

```
{
  "compileOnSave": false,
  "compilerOptions": {
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "module": "commonjs",
    "moduleResolution": "node",
    "noEmitOnError": true,
    "noImplicitAny": false,
    "removeComments": false,
    "sourceMap": true,
    "target": "es5"
  },
  "exclude": [
    "wwwroot"
  ]
}
```

**Comment by sky-coding** [comment]
I haven't seen this error on a similar setup (v2.1.1). Could you delete your `tsconfig.json` file and rebuild, to verify webpack is reading the correct config?

**Comment by goyaltushar92** [comment]
I was too having similar issue, but i find problem in my Webpack configuration
```javascript
module.exports = {
    entry: './UI/app.main.ts',
    output: {
        path: __dirname + '/wwwroot',
        filename: 'app.bundle.js',
        publicPath: '/'
    },
    devtool: "source-map",
    module: {
        loaders: [{
            test: /\.tsx?$/,
            include: '/UI',     // problem was due to this as my tsconfig.json was in root directory
            exclude: /node_modules/,
            loader: 'awesome-typescript-loader'
        }, {
                enforce: "pre",
                test: /\.js$/,
                loader: "source-map-loader"
        }]
    },
    resolve: {
        alias: { "@app": '/UI' },
        extensions:[ ".webpack.js", ".web.js", ".js", ".json", ".ts", '.tsx']
    }
};
```
